src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -230,6 +245,56 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator {
             await RespondAsync(request, ct, input);
         }
 
+        public Task<SendBlobResult> SendBlobAsync(byte[] data, CancellationToken ct) {
```

Comment:
They're semantically completely different. An eval message triggers an eval and returns the result; the input is an expression. A send blob message uploads a blob, and returns its ID; the input is a byte array.